modules/gitops-creating-a-new-client-using-keycloak.adoc

```diff
@@ -0,0 +1,45 @@
+[id="creating-a-new-client-in-keycloak_{context}"]
+= Creating a new client in Keycloak
+
+.Procedure
+
+. Create a new Argo CD Instance in the `argocd` namespace
++
+[source,yaml]
+----
+kubectl create -n argocd -f examples/argocd-keycloak.yaml
+----
+. After creating the Argo CD instance, login into the keycloak console.
++
+[source,yaml]
+----
+kubectl -n argocd get route keycloak
+
+NAME   HOST/PORT                                                                PATH   SERVICES   PORT    TERMINATION   WILDCARD
+keycloak    keycloak-default.apps.ci-ln-******.origin-ci-int-aws.dev.**.com          keycloak        <all>   reencrypt     None
+----
+. Get the Keycloak Pod Name which is stored as environment variables in the keycloak pod.
```

Comment:
please move this under "Logging into Keycloak" section.

Also there is a modification.
Get the Keycloak credentials which are stored as environment variables in the keycloak pod.